internal/host/statemachine.go

```diff
@@ -312,6 +312,24 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler) sta
 		PostTransition:   th.PostUnbindHost,
 	})
 
+	sm.AddTransition(stateswitch.TransitionRule{
+		TransitionType: TransitionTypeUnbindHost,
+		SourceStates: []stateswitch.State{
+			stateswitch.State(models.HostStatusPreparingForInstallation),
+			stateswitch.State(models.HostStatusPreparingFailed),
+			stateswitch.State(models.HostStatusPreparingSuccessful),
+			stateswitch.State(models.HostStatusInstalling),
+			stateswitch.State(models.HostStatusInstallingInProgress),
+			stateswitch.State(models.HostStatusInstallingPendingUserAction),
+			stateswitch.State(models.HostStatusResettingPendingUserAction),
+			stateswitch.State(models.HostStatusInstalled),
+			stateswitch.State(models.HostStatusResetting),
+			stateswitch.State(models.HostStatusAddedToExistingCluster),
+		},
+		DestinationState: stateswitch.State(models.HostStatusUnbindingPendingUserAction),
+		PostTransition:   th.PostUnbindHost,
+	})
```

Comment:
Since the host will continue sending state updates, you must make sure that it always stays in UnbindPedningUserAction, unless it has reregistered.  You can use Noop transactions (see in the statemachine.go file). 
Also, you must add transition from UnbindPedningUserAction to Discovering-Unbound in case node is rebooted with ISO

Reply:
Done